src/staking-v3/components/Edit.vue

```diff
@@ -22,16 +21,24 @@
         <div class="box__bottom">
           <span>{{ $t('stakingV3.editYourInfo.text') }}</span>
         </div>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('common.comingSoon') }}</span>
+        </q-tooltip>
       </div>
-      <div class="box--link">
+      <a
+        href="https://docs.astar.network/docs/use/dapp-staking/for-devs/create-promotion/"
```

Comment:
Let's import the url from `docsUrl`. You can find it at`src/links`.

Reply:
@impelcrypto Thank you! Fixed in [8117539](https://github.com/AstarNetwork/astar-apps/pull/1143/commits/8117539ff9503c8fd1782ea1252378031518755f)